templates/create_cloud_conf.sh

```diff
@@ -0,0 +1,111 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Provides tools for external cloud provider
+
+CAPO_SCRIPT=create_cloud_conf.sh
```

Comment:
why we define CAPO?

also this is not clear to me why we are adding this. can you give an explanation? maybe update the PR summary to add the reason we are adding this.

and add a readme to explain how to use that and what is the intent as well 

thanks so much!